keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/memory/InMemoryDeliveryConfigRepository.kt

```diff
@@ -91,6 +91,10 @@ class InMemoryDeliveryConfigRepository(
   ): ConstraintState? =
     constraints["$deliveryConfigName:$environmentName:$artifactVersion:$type"]
 
+  override fun getConstraintStateById(uid: String): ConstraintState? {
+    TODO("not implemented") // To change body of created functions use File | Settings | File Templates.
```

Comment:
nit: remove this comment please :)

Reply:
Sure. I actually need to implement the method, possibly it in tests.